src/Apps/Sell/Utils/previousSubmissionUtils.ts

```diff
@@ -0,0 +1,38 @@
+import { SellFlowStep } from "Apps/Sell/SellFlowContext"
+import createLogger from "Utils/logger"
+import { useEffect, useState } from "react"
+
+const SUBMISSION_ID_KEY = "previousSubmissionID"
+const SUBMISSION_STEP_KEY = "previousSubmissionStep"
+
+const logger = createLogger("previousSubmissionUtils")
+export const storePreviousSubmission = (
+  submissionID: string,
+  step: SellFlowStep
+) => {
+  localStorage.setItem(SUBMISSION_ID_KEY, submissionID)
+  localStorage.setItem(SUBMISSION_STEP_KEY, step)
+}
+
+export const usePreviousSubmission = () => {
+  const [submissionID, setSubmissionID] = useState<string | null>(null)
+  const [step, setStep] = useState<SellFlowStep | null>(null)
+
+  const getSubmission = async () => {
+    try {
+      const id = await localStorage.getItem(SUBMISSION_ID_KEY)
+      const step = await localStorage.getItem(SUBMISSION_STEP_KEY)
```

Comment:
I very much recommend using a Yup schema whenever you interact with localStorage in order to guarantee the data you receive. You'll avoid the casting below as well.

Reply:
Good idea! https://github.com/artsy/force/pull/14153